serve-site.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cd www
+
+# Generate hardware.inc.js
+echo "var hardware_inc = String.raw\`" > js/hardware.inc.js
```

Comment:
Just a note for when we'll have a bundler: hardware.inc can be imported as a git submodule as well: https://github.com/gbdev/hardware.inc/